easybuild/easyconfigs/r/RevBayes/RevBayes-1.1.1-GCC-10.2.0.eb

```diff
@@ -0,0 +1,40 @@
+# easybuild easyconfig
+#
+# John Dey <jfdey@fredhutch.org>
+# Fred Hutchinson Cancer Research Center - Seattle Washington - US
+easyblock = 'CmdCp'
+
+name = 'RevBayes'
+version = '1.1.1'
+
+homepage = 'https://revbayes.github.io/'
+
+description = """RevBayes provides an interactive environment for statistical computation in
+ phylogenetics. It is primarily intended for modeling, simulation, and Bayesian inference in
+ evolutionary biology, particularly phylogenetics."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/revbayes/revbayes/archive']
+sources = ['%(version)s.tar.gz']
+checksums = ['d61293fceac817d8203ed1e828661d76c73fa16bf04458a50a37057e99fd40c0']
+
+builddependencies = [('CMake', '3.18.4')]
+
+dependencies = [
+    ('Boost', '1.74.0'),
+    ('Java', '11', '', True),
```

Comment:
@fizwit is Java really needed for this software?

Reply:
Not sure where java came from, I have rebuilt without Java, and seems to work fine.